### PR TITLE
Improve installation docs

### PR DESCRIPTION
**Pull request by ntouran**
I changed the installation instructions to not require git in all cases. 
I also expanded the installation instructions a bit more and made some minor 
cross-links.

### CODE REVIEW

File: doc/user/inputs/index.rst [hunk 1]
```diff
@@ -5,6 +5,9 @@ Inputs
 ARMI input files define the initial state of the reactor model and tell ARMI what kind of analysis should be
 performed on it.
 
+.. note:: We have an :doc:`input walkthrough </tutorials/walkthrough_inputs>` tutorial for a quick 
+    overview of th inputs.
```
Comment (ntouran) [inline doc/user/inputs/index.rst:9]: the

File: doc/user/user_install.rst [hunk 2]
```diff
@@ -82,11 +81,15 @@ Then, install ARMI into your venv with::
 	In Windows, click *Start* and type ``Edit Environmental Variable`` to adjust ``PYTHONPATH``. 
 	In Linux, add ``export PYTHONPATH=/path/to/armi/source`` in a  user profile script (like ``.bashrc``).
 
-To see if it worked, run the ARMI launcher script::
+To see if it worked, run::
+
+    (armi-venv) $ python -m armi
```
Comment (jakehader) [inline doc/user/user_install.rst:86]: python3 to be consistent with other docs?

File: armi/utils/gridEditor.py [hunk 3]
```diff
@@ -6,6 +6,13 @@
 This provides a handful of classes which provide wxPython Controls for manipulating
 grids and grid Blueprints.
 
+Use
+===
+The grid editor may be invoked with the :py:mod:`armi.cli.gridGui` entry point::
+
+    $ python -m armi grids
```
Comment (jakehader) [inline armi/utils/gridEditor.py:13]: python3 to be consistent with the other part of the docs?

**Review by jakehader — APPROVED** [review]
Just change the `python` -> `python3` across the board and it's looks good.

**Comment by ntouran** [comment]
as it turns out, not all machines or installations of python make a `python3` command so that can be extra confusing. I may change it back to `python`.

**Pull request merged**